assets/booking_index.js

```diff
@@ -4,31 +4,42 @@ let formData = new FormData();
 let bookingId;
 let newBookingStartAt;
 let newBookingEndAt;
-moment.locale('fr');
-/* FullCalendar_integration */
 
 import { Calendar } from '@fullcalendar/core';
 import timeGridPlugin from '@fullcalendar/timegrid';
 import dayGridPlugin from '@fullcalendar/daygrid';
+import listPlugin from '@fullcalendar/list';
 import interactionPlugin from '@fullcalendar/interaction';
 import frLocale from '@fullcalendar/core/locales/fr';
 import moment from "moment";
 
+moment.locale('fr');
 
 document.addEventListener('DOMContentLoaded', function() {
 
+    function setBookingId(info){
+        bookingId = info.event.id
+        document.getElementById('bookingEdition').setAttribute('href', bookingId + '/edit')
+        document.getElementById('bookingOptionsTitle').innerText = moment(info.event.start).format('LLL');
+    }
+
+    function setBookingStartEndDateTime(info){
+        newBookingStartAt = info.startStr;
+        newBookingEndAt = info.endStr;
+    }
+
     let calendarEl = document.getElementById('calendar');
 
     let calendar = new Calendar(calendarEl, {
-        plugins: [ timeGridPlugin, dayGridPlugin,interactionPlugin],
+        plugins: [ timeGridPlugin, dayGridPlugin, interactionPlugin, listPlugin],
         locale:frLocale,
         timeZone: 'Europe/Paris',
         selectable: true,
         allDaySlot:false,
         headerToolbar: {
             start:'prev,next today',
             center: 'title',
-            end: 'dayGridMonth,timeGridDay',
+            end: 'dayGridMonth,timeGridDay,listYear',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/LeonardDeVanGogh/VosgesActivies/pullRequest?prid=7268173)
```suggestion
            end: "dayGridMonth,timeGridDay,listYear",
```